app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None | str:
+    file_names = command.split()
```

Comment:
use unpacking here
here is an example: `command, old_file, new_file = command.split()`